schema.graphql

```diff
@@ -289,3 +289,23 @@ type FarmHarvestEvent @entity {
   timestamp: BigInt!
   blockNumber: BigInt!
 }
+
+type SushiHarvestEvent @entity {
+  id: ID!
+  xSushiHarvested: BigInt!
+  totalxSushi: BigInt!
+  toStrategist: BigInt!
+  toGovernance: BigInt!
+  toBadgerTree: BigInt!
+  timestamp: BigInt!
+  blockNumber: BigInt!
+}
+
+type BadgerBalance @entity {
```

Comment:
This is an example of how not to use a graphQL entity. 

```
type TokenBalance @entity {
  id: ID!
  balance: BigInt!
  token : Token!
}

type Token @entity {
  id: ID!
  decimals: BigInt!
  name: String!
  symbol: String!
  totalSupply: BigInt!
}
```

Reply:
https://thegraph.com/docs/define-a-subgraph#bad-example

Reply:
This also applies to harvest events which can be made to do the same with generic token balances for amount harvested.